### PR TITLE
More splitting and spawn stuff

### PR DESCRIPTION
**Pull request by MeriKatt**

### CODE REVIEW

**Comment by MeriKatt** [comment]
2nd commit added winapi to the cargo.toml so that i could build lol
Fixed location and spawn file errors for names as well as fixed location names in checks file.

**Comment by bananaturtlesandwich** [comment]
What does it need winapi for? Is this a building from different os thing?

**Comment by bananaturtlesandwich** [comment]
Also I'm probably just gonna check the start tag stuff because that's where something might have slipped past and I trust you on the logic stuff

File: src/logic/spawns.rs [hunk 1]
```diff
@@ -1,46 +1,62 @@
...
+    ("dungeon1", L::CsPrisonEntry),
+    ("lowerWestSave", L::CsMain),
+    ("lowerWest", L::CsTheatreEntryNearPrison),
+    ("lowerSouthHigh", L::CsKeepEntryMain),
+    ("startGazebo", L::CsMain),
+    ("lowerNorth", L::CsKeepEntryRamp),
+    ("exterior1", L::CsBaileyEntry),
+    ("lowerMiddle", L::CsKeepClimbEntrance),
+    ("lowerEastSave", L::CsMain),
+    ("lower1", L::CsPrisonEntry),
```
Comment (bananaturtlesandwich) [inline src/logic/spawns.rs]: lower1 is from the dungeon side of the transition

File: src/logic/spawns.rs [hunk 2]
```diff
@@ -1,46 +1,62 @@
-use super::Location as L;
+use super::*;
+use Location as L;
 
-pub const SPAWNS: [(&'static str, L); 34] = [
+pub const SPAWNS: [(&'static str, L); 49] = [
+    //Prison starts
     ("gameStart", L::EarlyPrison),
     ("dungeonlowestSave", L::LatePrison),
```
Comment (bananaturtlesandwich) [inline src/logic/spawns.rs]: did you say you were splitting for the save after the first breakable wall and before the second for this dungeonlowestSave?

File: src/logic/spawns.rs [hunk 3]
```diff
@@ -1,46 +1,62 @@
...
+    ("exteriorSouthEast", L::EbEntryRuins),
+    ("exteriorEast", L::EbEntryUnderBelly),
+    // Underbelly starts
     ("cavesSouth", L::BaileyHole),
-    // same for keep entrance and save near heliacal power
     ("dungeonWest", L::PrisonHole),
-    ("cavesWestSave", L::MainUnderbelly),
-    ("postLightSave", L::MainUnderbelly),
-    ("cavesSouthSave", L::MainUnderbelly),
+    ("cavesWestSave", L::VAscendantLight), // Split AL due to DB reqs for levers / Doors/ mobs
```
Comment (bananaturtlesandwich) [inline src/logic/spawns.rs]: guess you can remove these comments now

**Review by bananaturtlesandwich — CHANGES_REQUESTED** [review]
only a few minor start tag things (again i trust you for logic)

**Pull request merged**